.github/workflows/charts-release.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
           #!/bin/bash
           CHARTS=(${{ steps.collect-changes.outputs.modifiedChartsAfterTag }})
           mkdir -p website/static/img/chart-icons || echo "chart-icons path already exists, continuing..."
-          rm -rf website/charts/*
+          rm -rf website/docs/charts/*/${chart}
```

Comment:
First my path was missing `docs` so it didn't do anything.
But, as I can understand this step only has changed charts?
If so, we can't nuke all docs but only the ones with matching chart name from all trains

Reply:
If this steps generates all docs and not changed only,
it can be changed to 
`website/docs/charts/*`